### PR TITLE
added .htaccess

### PR DESCRIPTION
**Pull request by PeriscopeProject**
Request of permanent link for European Periscope Project website, ontology and web tool.

### CODE REVIEW

File: periscope/.htaccess [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Header set Access-Control-Allow-Origin *
+Header set Access-Control-Allow-Headers DNT,X-Mx-ReqToken,Keep-Alive,User-Agent,X-Requested-With,If-Modified$
+Options +FollowSymLinks
+RewriteEngine on
+# Rewrite rule to serve the Periscope Project atlas tool
+RewriteRule ^$ https://atlas.periscopeproject.eu/pewg/ [R=302,L]
+# Rewrite rule to serve the Periscope Project website
+RewriteRule ^$ https://periscopeproject.eu/ [R=302,L]
+# Rewrite rule to serve the Periscope Project ontology
+RewriteRule ^$ https://periscope.lst.tfo.upm.es/ [R=302,L]
```
Comment (davidlehn) [inline periscope/.htaccess:10]: These rules are all using the same pattern so only the first will match.

File: periscope/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# BBS Vocabulary
```
Comment (davidlehn) [inline periscope/README.md:1]: It looks like this whole file was just copied from elsewhere?  Please update.

**Comment by PeriscopeProject** [comment]
removed README file

**Comment by dgarijo** [comment]
@PeriscopeProject please add a readme file stating who the maintainer for this w3id will be, along with a short description (1 line)

**Comment by davidlehn** [comment]
Can you add the github id for the maintainer(s).  We use that to make sure future PRs are ok to apply.  If it's an id other than PeriscopeProject, we'll have to ask about it.  Also please address my comment about the rules.  Thanks.

**Pull request closed without merging**